src/containers/UserSettings/UserSettings.tsx

```diff
@@ -32,9 +33,17 @@ export const UserSettings = ({settings: userSettings}: UserSettingsProps) => {
                                                 </Settings.Item>
                                             );
                                         }
+                                        const {description, ...rest} = setting;
                                         return (
-                                            <Settings.Item key={setting.title} {...setting}>
+                                            <Settings.Item
+                                                key={setting.title}
+                                                align="top"
+                                                {...rest}
+                                            >
                                                 <Setting {...setting} />
+                                                <Text color="secondary" as="div">
+                                                    {description}
+                                                </Text>
```

Comment:
A margin between setting control and text is needed

<img width="603" alt="Screenshot 2025-01-16 at 17 39 27" src="https://github.com/user-attachments/assets/26f37d12-254e-4c67-bcf5-c6d89a14d1de" />

Reply:
I'm not sure... I've looked at navigation stories, and there's no additional space between control and text:  https://preview.yandexcloud.dev/navigation/?path=/story/components-settings--showcase (see "Communication" tabl)